app.js

```diff
@@ -1,7 +1,16 @@
 "use strict";
 
+import maplibregl from "maplibre-gl";
+import "maplibre-gl/dist/maplibre-gl.css";
+import geojsonExtent from "@mapbox/geojson-extent";
+import MapboxDraw from "@mapbox/mapbox-gl-draw";
+import "@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw.css";
+import { RouteSnapper, fetchWithProgress } from "route-snapper/lib.js";
+
+// Do this after mapbox-gl-draw.css to override some stuff
+import "./css/map_controls.css";
```

Comment:
This is a little magic to me still. If you want some CSS included in a web page, you can just import it in a `.js` and vite does some kind of bundling magic. It just works.

Reply:
Nice!